azkaban-exec-server/src/main/java/azkaban/execapp/Trigger.java

```diff
@@ -0,0 +1,94 @@
+package azkaban.execapp;
+
+import azkaban.trigger.Condition;
+import azkaban.trigger.TriggerAction;
+import java.util.Iterator;
+import java.util.LinkedList;
+import java.util.List;
+import org.apache.log4j.Logger;
+
+
+public class Trigger implements Runnable {
```

Comment:
+1. These classes seem to have similar data. We can use one to wrap the other. Or have a single Trigger class and have 'controller' kind of classes to work with these objects. thoughts?